src/components/input/theme.ts

```diff
@@ -1,4 +1,7 @@
+import { DefaultTheme } from '@react-navigation/native';
+
 const theme = {
```

Comment:
```suggestion
const theme: Theme = {
```

Reply:
fixed in [e4ed6df](https://github.com/BinaryStudioAcademy/bsa-winter-2022-growup-mobile/pull/16/commits/e4ed6df525c133157c5109fd79f7db8fbaeb773b)